src/main/java/com/exactpro/th2/check1/Check1Main.java

```diff
@@ -42,10 +43,11 @@ public static void main(String[] args) {
             GrpcRouter grpcRouter = commonFactory.getGrpcRouter();
             Check1Configuration configuration = commonFactory.getCustomConfiguration(Check1Configuration.class);
 
-            CollectorService collectorService = new CollectorService(messageRouter, commonFactory.getEventBatchRouter(), configuration);
+            ResultsStorage resultsStorage = new ResultsStorage(configuration.getResultsCleanupTimeoutSec());
```

Comment:
I think we should use the same parameter that we use for rule clean up. Don't see any reason to add additional parameter for that